### PR TITLE
Reload styling on theme change

### PR DESCRIPTION
**Pull request by lchanmann**
## Ref
- https://nanopay.atlassian.net/browse/NP-4920

## Issues
- customCss expansion is broken when changing theme

## Changes
- Add themeChange event
- Reload application styles when theme changes

### CODE REVIEW

**Comment by lchanmann** [comment]
@kgrgreer @jlhughes @Nauna @nanopay-arthur Please review when you have a chance, thanks.

File: src/foam/nanos/controller/ApplicationController.js [hunk 1]
```diff
@@ -412,6 +423,22 @@ foam.CLASS({
         self.onUserAgentAndGroupLoaded();
         self.mementoChange();
       });
+
+      // Reload styling on theme change
+      this.onDetach(this.sub('themeChange', () => {
+        for ( const eid in this.styles ) {
+          let text = this.styles[eid];
+          for ( var i = 0 ; i < this.MACROS.length; i++ ) {
```
Comment (sarthak-marwaha) [inline src/foam/nanos/controller/ApplicationController.js]: Couldn't this part use the `returnExpandedCSS()` function that is in applicationController?

**Comment by jlhughes** [comment]
LGTM

File: src/foam/nanos/controller/ApplicationController.js (hunk 1 above)
Comment (lchanmann) [reply src/foam/nanos/controller/ApplicationController.js]: Fixed in https://github.com/kgrgreer/foam3/pull/589/commits/1b89b36fb9c14974dc80dc8d456a1f4b87e12b94.

**Review by sarthak-marwaha — APPROVED** [review]
LGTM

**Pull request merged**